user/enterprise/ssl-certificate-management.md

```diff
@@ -88,131 +88,167 @@ Upon restart you can then verify whether SSL verification is now working. If you
 
 ## Using a Let's Encrypt SSL Certificate
 
-Travis CI Enterprise works well together with a Let's Encrypt SSL Certificate. To obtain one for your domain, please follow the steps below.
+You can use a certificate from [Let's Encrypt](https://letsencrypt.org/) instead of a self-signed certificate or a certificate purchased from a trusted certificate authority. Certificates from Let's Encrypt are free and behave the same as those purchased from a trusted certificate authority. More information about Let's Encrypt can be found [here](https://letsencrypt.org/about/).
 
-What you need:
+What you will need:
 
-1. An email address Let's Encrypt can send emails to. This will be used to notify about urgent renewal and security notices.
+1. An email address (Let's Encrypt will send notifications regarding urgent renewal and security issues).
```

Comment:
```suggestion
- An email address (Let's Encrypt will send notifications regarding urgent renewal and security issues).
```

*these ones* don't need to be ordered :-p

Reply:
I just went on a roll and couldn't stop! 😄

Reply:
Exactly!

Reply:
Addressed in 192be263